exercises/reverse-string/canonical-data.json

```diff
@@ -0,0 +1,39 @@
+{
+  "exercise": "reverse-string",
+  "version": "1.0.0",      
+  "comments": [
+        "If property based testing tools are available, a good property to test is reversing a string twice: reverse(reverse(string)) == string"
+      ],
+  "cases": [
+    {
+      "description": "empty string",
+      "property": "reverse",
+      "input": "",
+      "expected": ""
+    },
+    {
+      "description": "reverse a string",
+      "property": "reverse",
+      "input": "robot",
+      "expected": "tobor"
+    },
+    {
+      "description": "reverse a string with a capital letter",
```

Comment:
remove "reverse a"

what about "a capitalized word"

Reply:
I like this suggestion.